src/libraries/System.Net.Sockets/tests/FunctionalTests/Connect.cs

```diff
@@ -109,6 +109,8 @@ public async Task Connect_DualMode_MultiAddressFamilyConnect_RetrievedEndPoints_
         }
 
         [Fact]
+        [ActiveIssue("https://github.com/dotnet/runtime/issues/54677", TestPlatforms.AnyUnix)]
+        [ActiveIssue("https://github.com/dotnet/runtime/issues/55709", TestPlatforms.AnyUnix)]
```

Comment:
Did these ever fail on macOS, or only on Linux?

Reply:
Only on Linux, but I didn't find better TestPlatforms ... is there one?

Reply:
TestPlatforms.Linux ?